bfscripts.spec.rpkg

```diff
@@ -0,0 +1,56 @@
+%global debug_package %{nil}
+
+Summary: Useful scripts for managing Mellanox BlueField hardware.
+Name: mlxbf-bfscripts
+Version: 3.0.alpha3
+Release: 1%{?dist}
+License: BSD
+
+Source: {{{ git_dir_pack dir_name=bfscripts source_name=mlxbf-bfscripts.tar.gz }}}
```

Comment:
Do we have issues with the rpmlint tool for source RPMs generated with such syntax?

Reply:
The `{{{ }}}` syntax is used by rpkg-util (https://pagure.io/rpkg-util) to generate parts of the spec file. It is not present in the SRPMs. rpmlint does have issues with the spec file, but I'm going to fix those in a separate commit.